src/components/Button/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Alert, Button } from 'react-native';
+import { ContainerButton } from './styles';
+
+export default function ButtonMain({ children }) {
```

Comment:
@HigorSantosCIC aqui acho que podemos colocar um pouco mais de propriedades, talvez a cor do botão e as demais propriedades desses, por isso é interessante passar um ...rest que seria o restante dos argumentos.
```suggestion
export default function ButtonMain({ children, ...rest}) {
```